v5/driver/server.go

```diff
@@ -89,18 +92,30 @@ func main() {
 			}
 
 			var configMap map[string]string
-
 			json.Unmarshal(*setup.Options.Configuration, &configMap)
 
-			switch setup.Options.Env {
-			case "localhost":
-				os.Setenv("ROLLOUT_MODE", "LOCAL")
-			case "qa":
-				os.Setenv("ROLLOUT_MODE", "QA")
-			default:
-				os.Setenv("ROLLOUT_MODE", "")
+			var networkConfig model.NetworkConfigurationsOptions = nil
+
+			if (setup.Options.Env == "container") {
+				networkConfig = client.NewNetworkConfigurationsOptions(client.NetworkConfigurationsBuilder{
+					GetConfigApiEndpoint:     configMap["CD_API_ENDPOINT"],
+					GetConfigCloudEndpoint:   configMap["CD_S3_ENDPOINT"],
+					SendStateApiEndpoint:     configMap["SS_API_ENDPOINT"],
+					SendStateCloudEndpoint:   configMap["SS_S3_ENDPOINT"],
+					AnalyticsEndpoint:        configMap["ANALYTICS_ENDPOINT"],
+					PushNotificationEndpoint: configMap["NOTIFICATIONS_ENDPOINT"],
+				})
+			} else {
+				switch setup.Options.Env {
+				case "localhost":
+					os.Setenv("ROLLOUT_MODE", "LOCAL")
+				case "qa":
+					os.Setenv("ROLLOUT_MODE", "QA")
```

Comment:
do we need `test` in here as well now?

Reply:
inside the SDK there's no `TEST` for the envvar `ROLLOUT_MODE`
we decided not to put new hard coded values here AFAIK
it should only be dynamic values, which is what `container` indicate
we will use the roxOptions, and not the env variable anyway

so there's no point in setting it for the e2e driver..